app/cdap/text/text-en.yaml

```diff
@@ -3432,8 +3432,22 @@ features:
       success: You have successfully created the pipeline "{pipelineName}".
     Skip: Skipped
   WranglerNewUI:
+    BreadCrumb:
+      labels:
+        dataSources: Data Sources
     NoRecordScreen:
       gridTable:
-        title: No rows left in this sample
         subtitle: Remove some recipe steps to view some data
+        title: No rows left in this sample
+    Snackbar:
```

Comment:
Strings should be under their feature areas, not `Snackbar`

Reply:
Done